src/app/modals/add-to-group/add-to-group.component.scss

```diff
@@ -59,27 +59,6 @@
         .wrapper {
             display: inline-flex;
 
-            .help-link {
```

Comment:
This block of css also appears in _modals.scss. Removed it here and moved it to the top level app.scss